test/ui-test/testSuites/suite_wallet/shared/steps/walletSteps.py

```diff
@@ -85,4 +100,12 @@ def step(context, account_name: str):
     
 @Then("the transactions are listed for the added account")
 def step(context):
-    _walletScreen.verify_transactions_exist()
\ No newline at end of file
+    _walletScreen.verify_transactions_exist()
+    
+@Then("the saved address \"|any|\" has favourite status \"|any|\"")
+def step(context, name, favourite):
+    _walletScreen.check_favourite_status_for_saved_address(name, str_to_bool(favourite))
+    
+###########################################################################
+### COMMON methods used in different steps given/when/then region:
+########################################################################### 
```

Comment:
not a fan of those comment section, reason is that very quickly someone will not follow it and they will just be useless, but I am fine having them

Reply:
If there is a better or more standard way to do it in `Python`, I'm ok of changing it like this, but I think we need a way to have these files organized and with some notes to know the options we have since there is a lot of people that can potentially work on it, and we need to try to control the "potential mess". I understand your point since if it is not followed, it's useless.

Open to suggestions! :)